js/render/painter.js

```diff
@@ -152,7 +152,7 @@ Painter.prototype.setup = function() {
     gl.bufferData(
         gl.ARRAY_BUFFER,
         new Int16Array([
-            0, 0, EXTENT - 1, 0, EXTENT - 1, EXTENT - 1, 0, EXTENT - 1, 0, 0]),
+            0, 0, EXTENT, 0, EXTENT, EXTENT, 0, EXTENT, 0, 0]),
```

Comment:
I'm not sure. I think conceptually with the -1 the lines are always strictly within a tile's boundaries. But since gl.LINES are snapped to the nearest pixel this isn't actually useful. -native doesn't have this

With the -1 the tile boundaries sometimes overlap with neighbours (creating a 1px line) and sometimes doesn't overlap (creating a 2px line). This makes it a consistent 1px.